pkg/lib/patterns.js

```diff
@@ -397,16 +397,15 @@ $.fn.update_privileged = function update_privileged(perm, denied_message, placem
 
         tooltip_element.tooltip(options);
 
-        if ($(this).hasClass("disabled") === allowed) {
-            $(this).toggleClass("disabled", !allowed);
-            tooltip_element.attr('data-original-title', null);
+        $(this).toggleClass("disabled", !allowed);
```

Comment:
Ah, so that's the 3v1l function that uses the "disabled" class? But it has done that for a long time already, right? Was it just that our test cases previously didn't encounter elements with the disabled class?

Reply:
Correct.

Reply:
Resolving, this was fixed with the `update_privileged()` work and fell out of this PR. Nice!